api/api.yaml

```diff
@@ -0,0 +1,218 @@
+                    {
+                      "id": 3,
+                      "title" : "added new todos",
+                      "note" :  "go test the post",
+                      "due_date" :  "2019-02-05T06:51:36.749Z"
+                      "created_by": "2019-02-05T06:51:36.749Z"
+                    }
+                ]
+               }
+            }
+          schema:
+                $ref: '#/definitions/todosObjecttinDB'
+                
+        '400':
+          description: 'invalid input, object invalid'
+          examples:
+            application/json:
+              {
+                "error": {
+                  "message": "invalid input"
+                }
+              }  
+          schema:
+            $ref: '#/definitions/errorObject'
+        '404':
+          description: Error the provided todoid is not found.
+          examples:
+              application/json:
+                {
+                  "error": {
+                    "message": "todos with id <todos:id> was not found."
+                  }
+                }
+          schema:
+              $ref: '#/definitions/errorObject'
+/todos:
+    'post':
+      summary: Creates todos 
+      description: Used to create the todos 
+      produces: 
+          - application/json
+      parameters:
+        - name: todos
+          description: An object containing all the required details
+          in: body
+          schema:
+            $ref: '#/definitions/todosObject'
+    responses:
+        '201':
+          description: todos created
+          examples:
+          application/json:
+          {
+            data: {
+                "todos": [
+                {
+                  "id": 3,
+                  "title" : "added new todos",
+                  "note" :  "go test the post",
+                  "due_date" :  "2019-02-05T06:51:36.749Z"
+                  "created_by": "2019-02-05T06:51:36.749Z"
+                }
+                ]
+
+            }
+          }
+           
+          schema:
+             $ref: '#/definitions/todosobjectinDB'
+        '400':
+          description: 'invalid input, object invalid'
+          examples:
+            application/json:
+              {
+                "error": {
+                  "message": "invalid input"
+                }
+              }
+          schema:
+             $ref: '#/definitions/errorsObject'
+
+
+    'get':
+     summary: Retrieves todos details
+     description: provides all the required todos that have been created 
+     parameters:
+        - name: todos
+          in: query
+          required: true 
+          type: string 
+     responses:
+        '200':
+          description: Array of Todo model instances
+          examples:
+            application/json:
+              schema:
+                $ref: '#/definitions/todosObjectinDB '
+        examples:
+            application/json:
+            {
+              data: {
+                    "todos": [
```

Comment:
Take a look athe api standards, it should not be data: {
todods: []}  ut just  {data: []}